modules/upload/client-react/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Route, NavLink } from 'react-router-dom';
+import { translate, TranslateFunction } from '@restapp/i18n-client-react';
+import { MenuItem } from '@restapp/look-client-react';
+import ClientModule from '@restapp/module-client-react';
+
+import Upload from './containers/Upload';
+import resources from './locales';
+import uploadReducer from './reducers';
+
+const NavLinkWithI18n = translate('upload')(({ t }: { t: TranslateFunction }) => (
```

Comment:
The `index` file should be clean and doesn't contain any components.

Reply:
Fixed